packages/clay-core/src/picker/Picker.tsx

```diff
@@ -382,6 +382,16 @@ export function Picker<T extends Record<string, any> | string | number>({
 		);
 	}
 
+	let menuMinWidth = '160px';
+	let menuWidth = 'auto';
+
+	if (typeof width === 'number') {
+		menuMinWidth = `${triggerRef.current?.clientWidth || 0}px`;
+		menuWidth = `${width}px`;
+	} else if ((triggerRef.current?.clientWidth || 0) > 160) {
+		menuMinWidth = `${triggerRef.current?.clientWidth}px`;
```

Comment:
I'm guessing this should also be `triggerRef.current?.clientWidth || 0`

Reply:
I don't think we need to add `|| 0` because the condition has already ensured that triggerRef.current?.clientWidth is greater than 160.
What do you think?
```javascript
else if ((triggerRef.current?.clientWidth || 0) > 160)
```

Reply:
That makes sense to me. It's probably worth creating a variable for `triggerRef.current?.clientWidth` rather than checking that 3 different times.

Reply:
Done!